### ISSUE TITLE
Link/tab missing.

### ISSUE THREAD

**Issue by dagroll** [MicrosoftDocs/OfficeDocs-Enterprise#134]
The "Update your firewall's outbound allow lists" has the following sentence:
The easiest way to choose which network requests to trust is to use our pre-built PAC files on the Proxies tab above.  
However, the article does not have Proxies tab.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8762d73c-32fc-a4f9-162a-b81ae4052ea3
* Version Independent ID: 554cc99d-b929-8db0-7ab7-0f666438e8cf
* Content: [Managing Office 365 endpoints](https://docs.microsoft.com/en-us/office365/enterprise/managing-office-365-endpoints?redirectSourcePath=%252fen-us%252farticle%252fmanaging-office-365-endpoints-99cab9d4-ef59-4207-9f2b-3728eb46bf9a#ID0EADAAA=4._Web_service&ID0EACAAA=4._Web_service)
* Content Source: [Enterprise/managing-office-365-endpoints.md](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/blob/live/Enterprise/managing-office-365-endpoints.md)
* Service: **o365-administration**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by denisebmsft** [comment]
@dagroll thank you for the feedback. We're looking into this.

**Comment by AndreaBarr** [comment]
@dagroll Thank you for this feedback

I want to make sure this proxies tab issue is addressed so I am making sure that the Enterprise team gets this so it's can be investigated. 

I was able to get this into the backlog for the Enterprise team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
